lib/src/cupertino/field/cupertino_typeahead_field.dart

```diff
@@ -22,7 +22,7 @@ import 'package:flutter_typeahead/src/utils.dart';
 class CupertinoTypeAheadField<T> extends StatefulWidget {
   /// Called with the search pattern to get the search suggestions.
   ///
-  /// This callback must not be null. It is be called by the TypeAhead widget
+  /// You must have to specify either [suggestionsCallback] or [suggestionsLoadMoreCallback], but not both. It is be called by the TypeAhead widget
```

Comment:
nit: wording it more like this might be preferable:
```suggestion
  /// Either [suggestionsCallback] or [suggestionsLoadMoreCallback] must be specified. It is be called by the TypeAhead widget
```